Makefile

```diff
@@ -0,0 +1,40 @@
+
+release: pysense pysense2 
+
+
+pysense:
+	mkdir pysense
+	mkdir pysense/lib
+	# sensors
+	cp shields/lib/LIS2HH12.py pysense/lib/
+	cp shields/lib/LTR329ALS01.py pysense/lib/
+	cp shields/lib/MPL3115A2.py pysense/lib/
+	cp shields/lib/SI7006A20.py pysense/lib/
+	# pycoproc
+	cp shields/lib/pycoproc_1.py pysense/lib/pycoproc.py
```

Comment:
the problem, when we rename it is, that we also need to adapt the imports ... which we could do with sed, but ugh ... more and more magic in this 'release' process that needs maintaining. I'm not sure either way....